src/ServiceControl.Config/UI/Shell/ShellViewModel.cs

```diff
@@ -91,7 +91,7 @@ protected override void OnInitialize()
             RefreshInstances();
         }
 
-        void RefreshInstances()
+        public void RefreshInstances()
```

Comment:
I guess this was needed because we no longer dispatch over the event broker?

Reply:
Commented above.